staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation_test.go

```diff
@@ -85,7 +85,7 @@ func TestValidateCustomResourceDefinition(t *testing.T) {
 		{
 			name: "webhookconfig: invalid port 0",
 			resource: &apiextensions.CustomResourceDefinition{
-				ObjectMeta: metav1.ObjectMeta{Name: "plural.group.com"},
+				ObjectMeta: metav1.ObjectMeta{Name: "plural.group.com", Tenant: "test-te"},
```

Comment:
nit: extract "test-te" into a variable

Reply:
and do we need a case for system tenant

Reply:
As suggested, I extracted "test-te" into a string variable.

I don't think we need test cases for system tenant here. These tests are validating the  CRD-related structures and schemas, which are same for regular tenants and system tenants.